lib/ansible/playbook/__init__.py

```diff
@@ -75,7 +75,7 @@ def __init__(self,
         diff             = False,
         any_errors_fatal = False,
         vault_password   = False,
-        force_handlers   = False,
+        force_handlers   = C.DEFAULT_FORCE_HANDLERS,
```

Comment:
Sorry, I was just matching the others like the become family, forks, etc, which have the default both in the parser level and then here again. Maybe that's just for historical reasons?